src/main/create-operator-json/operators/mozzie.js

```diff
@@ -8,8 +8,7 @@ import r6operators from "r6operators";
 let mozzie = new Operator(r6operators.mozzie, operatorId.mozzie, "Operation Burnt Horizon");
 
 mozzie.addCounterNode(operatorId.twitch, counterType.hard, "Mozzie's Pests can hack Twitch's Shock Drone. The hacked drone can still shoot darts.");
-mozzie.addCounterNode(operatorId.flores, counterType.hard, "Mozzie's Pests can hack Flores' RCE-RATERO drone. The hacked drone cannot be controlled though.");
+mozzie.addCounterNode(operatorId.flores, counterType.hard, "Mozzie's Pests can hack Flores' RCE-RATERO drone. The hacked drone cannot be controlled, but will get destroyed harmlessly.");
```

Comment:
I'm not sure I understand this one. I don't really know what happens in game, but would "but will inevitably get destroyed." be accurate?

Reply:
Ah I see! Maybe "disabled" if it doesn't disappear?

If it does get destroyed, then "destroy it without detonating" makes sense 👍